flash_cards/views.py

```diff
@@ -1,3 +1,6 @@
 from django.shortcuts import render
-
+from django.contrib.auth.decorators import login_required, user_passes_test 
 # Create your views here.
+
+def index(request):
+    return render(request, "flash_cards/homepage.html")
```

Comment:
make sure we have a newline at the end of each python file (python doesn't like it when we don't!)
```suggestion
    return render(request, "flash_cards/homepage.html")
    
```

Reply:
Are you adding the template later? I don't see the templates folder in this Pull Request